experimental/framework/tinylicious-client/src/TinyliciousClient.ts

```diff
@@ -136,14 +136,15 @@ export class TinyliciousClientInstance {
  */
 // eslint-disable-next-line @typescript-eslint/no-extraneous-class
 export class TinyliciousClient {
-    private static globalInstance: TinyliciousClientInstance | undefined;
+    protected static globalInstance: TinyliciousClientInstance | undefined;
 
     static init(serviceConnectionConfig?: TinyliciousConnectionConfig) {
         if (TinyliciousClient.globalInstance) {
             console.log(
                 `TinyliciousClient has already been initialized. Using existing instance of
                 TinyliciousClient instead.`,
             );
+            return;
```

Comment:
I still think we should throw here.  This at least lets the customer blow up in a predictable way - there's no way to know if this call actually produced the config you were expecting or is still using a stale config.

Reply:
As per team decision, we will remove static from TinyliciousClient, so this implementation will be replaced by a constructor.